crates/hyperqueue/src/dashboard/ui/screens/worker/screen.rs

```diff
@@ -0,0 +1,129 @@
+use std::time::SystemTime;
+use termion::event::Key;
+
+use crate::dashboard::ui::screen::Screen;
+use crate::dashboard::ui::styles::{style_header_text, style_overview_footer};
+use crate::dashboard::ui::terminal::DashboardFrame;
+use crate::dashboard::ui::widgets::text::draw_text;
+
+use crate::dashboard::data::DashboardData;
+use crate::dashboard::ui::screen::controller::ScreenController;
+use crate::dashboard::ui::screens::worker::worker_info_table::WorkerInfoTable;
+use tako::WorkerId;
+use tui::layout::{Constraint, Direction, Layout, Rect};
+
+#[derive(Default)]
+pub struct WorkerOverviewScreen {
+    /// The worker info screen shows data for this worker
+    worker_id: WorkerId,
```

Comment:
Let's store this as `Option<WorkerId>`. With the current implementation, 0 will be used as a default, and if the screen is rendered in such state, it will draw some garbage or crash in some random place. If an `Option` is here, if the screen is drawn with the option being `None`, we can react to it explicitly (either by crashing or by switching the state to the overview screen).